core/src/main/scala/org/ensime/core/RichPresentationCompiler.scala

```diff
@@ -145,21 +145,6 @@ trait RichCompilerControl extends CompilerControl with RefactoringControl with C
   def askReloadAndTypeFiles(files: Iterable[SourceFile]) =
     askOption(reloadAndTypeFiles(files))
 
-  def askUsesOfSymAtPos(pos: Position)(implicit ec: ExecutionContext): Future[List[RangePosition]] = {
```

Comment:
I don't completely understand. We wouldn't  get anything extra using this, would we?
Since now we store `FullyQualifiedReference` references on different lines are different and we should get all of them.